repo/packages/N/nifi/700/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "packagingVersion": "4.0",
+  "upgradesFrom": [],
+  "downgradesTo": [],
+  "minDcosReleaseVersion": "1.12",
+  "name": "nifi",
+  "version": "1.0.0-1.9.2",
+  "maintainer": "support@mesosphere.io",
+  "description": "Apache NiFi",
+  "selected": false,
```

Comment:
I think we want to mark this true to signify we now `certify` the framework. There was some extra changes needed in other repos to propogate this. 

@takirala @vibhujain @ankitcid which projects did we need to update to get this to work for Hive?

Reply:
Yes @samvantran, Thank you for finding the conversation, and as you mentioned [in the review comment](https://github.com/mesosphere/universe/pull/2437#pullrequestreview-315214500), we need to follow those steps in order to make `nifi` as a certified package with appropriate whitelisting of it's package in universe-builder plugins.

As now I'm watching the changes done by @porridge re your feedback, the CI will mark checks as green when `universe-checker` and `universe-package-builder` will be merged, and when flip of `selected: true` will be done. (And those green checks will mark that we are good to go)